### ISSUE TITLE
Current enom prices

### ISSUE THREAD

**Issue by yarantoos** [ducohosting/whmcs-enom-updater#18]
It good idea to show current enom prices fetched.

Currently when we fetch enom prices , it shows nothing . I want to check new enom prices in whmcs and then update the prices if we need .

**Comment by cascer1** [comment]
Basically you would like to see a page containing the cached eNom prices somewhere?

**Comment by yarantoos** [comment]
Yes exactly .  . I  like to see a page containing the cached eNom prices

**Comment by yarantoos** [comment]
The module have a feature with fetching new prices from enom but we need to see changes and then update the prices . 
There is a good idea to see a page containing the cached eNom prices and current enom prices. and we can choose which tld we need to update.

**Comment by cascer1** [comment]
Thank you for expanding on the request. We'll see when we can implement this feature.
